client/game_logic.py

```diff
@@ -235,3 +235,10 @@ def load_state(self):
         if self.package_parsed[4] == '01':
             self.end = True
 
+
+if __name__ == '__main__':
+    # jakis_lock = threading.Lock()
+    # game = Game([], jakis_lock)
+    # game.start()
+    gv = GameView()
+    gv.start()
```

Comment:
Not used in normal application. Exists only for testing purpose. I'd suggest to remove it

Reply:
Done